app/src/androidTest/java/com/github/se/cyrcle/ui/report/AdminScreenTest.kt

```diff
@@ -156,4 +176,65 @@ class AdminScreenTest {
     composeTestRule.onNodeWithTag("ReportCard0").performClick()
     composeTestRule.onNodeWithTag("CheckReportsButton0").performClick()
   }
+
+  @Test
+  fun testAlertDialogAppearsOnMoreOptionsClick() {
+    composeTestRule.setContent {
+      AdminScreen(
+          navigationActions = navigationActions,
+          reportedObjectViewModel = reportedObjectViewModel,
+          parkingViewModel = parkingViewModel,
+          reviewViewModel = reviewViewModel)
+    }
+    composeTestRule.onNodeWithTag("ReportCard0").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNodeWithTag("MoreOptionsButton0").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNode(isDialog()).assertIsDisplayed()
+    composeTestRule.onNodeWithText("Details for this Reported Object:").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Loading parking details...").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Go To Parking").assertIsDisplayed().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").assertIsDisplayed().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").performClick()
```

Comment:
I suggest you `assertHasClickAction` and `assertIsDisplayed` before performing a click since it makes the test more complete and robust.